test/FsAutoComplete.Tests.Lsp/FsAutoComplete.Tests.Lsp.fsproj

```diff
@@ -42,5 +41,12 @@
           DestinationFolder="$(OutputPath)"
           ContinueOnError="false" />
   </Target>
+
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Copy SourceFiles="$([System.IO.Directory]::GetParent($(BundledRuntimeIdentifierGraphFile)))\NuGet.Frameworks.dll"
+          DestinationFolder="$(OutputPath)"
+          ContinueOnError="false" />
+  </Target>
```

Comment:
I need to comment why this is needed.  short version here is that the `dotnet test` framework needs _a_ Nuget.Frameworks.dll, but in general we want to bind to the one from the SDK (because like MSBuild it needs to be runtime-bound).